charts/rancher-logging/0.0.1/charts/fluentd/templates/rolebinding.yaml

```diff
@@ -0,0 +1,22 @@
+
+{{- if .Values.podSecurityPolicy.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: RoleBinding
```

Comment:
It relates to `pod-security-policy.yaml`, merging into `pod-security-policy.yaml`.